tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -169,11 +327,10 @@ std::vector<std::shared_ptr<T>> createMeshFromPath(const std::string& path,
 
   // Issue #38 fix: as part of the post-processing, we remove all other components in file but
   // the meshes, as anyway the resulting shapes:Mesh object just receives vertices and triangles.
+  // John Wason Dec 2020 - Load more information
```

Comment:
Should we update to include vertices, triangles, normals colors and textures?

Reply:
Should the commented out aiComponent_NORMALS be removed?

Reply:
The property `AI_CONFIG_PP_RVC_FLAGS` is a mask flag, so if the flag is present, it disables that feature from being loaded. I am going to modify this section to set the flags based on the options you suggested in the other comment.

Reply:
This section has been updated to use the additional `normals`, `vertex_colors`, and `material_and_texture` arguments.